dev/AppLifecycle/ExtensionContract.h

```diff
@@ -15,14 +16,15 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
     {
```

Comment:
Don't use `static` for global variables in header files. This causes each client of the header file to get a separate copy of the variable. Use `constexpr` and/or `inline` to allow variables to be shared.